salt/metalk8s/metalk8s-ui/deployed.sls

```diff
@@ -31,3 +46,16 @@ Create metalk8s-ui service:
         type: NodePort
   require:
     - pkg: Install Python Kubernetes client
+
+Create ui-api-configmap ConfigMap:
+  kubernetes.configmap_present:
+    - name: ui-api-configmap
+    - namespace: kube-system
+    - kubeconfig: {{ kubeconfig }}
+    - context: {{ context }} 
+    - data:
+        config.json: |
+          {"ip": "{{ control_plane_ip }}", "port": "6443"}
```

Comment:
Let's not do IP/port things. This should be a URL, and used as an URL in the JS code.

Reply:
ok for url => debt ticket